recipe/meta.yaml

```diff
@@ -26,10 +26,13 @@ requirements:
     build:
         - python  # [win]
         - msinttypes  # [win]
+    #run:
+        #- libgcc  # [unix]
```

Comment:
I am never sure about this one. In this case the library links to `libgcc_s.so.1 (lib/libgcc_s.so.1)` and `libstdc++.so.6 (lib/libstdc++.so.6)` and those are provided by the `libgcc` package.

(See https://github.com/conda-forge/geos-feedstock/pull/10#issue-146018524)